src/locales/en/en.json

```diff
@@ -840,51 +840,90 @@
     },
     "alert": {
       "title": "Would you like to grant permission for using your camera?"
-    }
+    },
+    "pastemeerkatid": "Paste Meerkat ID"
   },
   "chat": {
     "tab": {
       "header": "Under development"
     }
   },
   "menu": {
-    "tab": {
-      "header": "Menu",
-      "items": {
-        "profile": "Profile",
-        "crypto": "Crypto",
-        "connections": "Connections",
-        "connectwallet": "Connect Wallet",
-        "lock": "Lock",
-        "cip": "CIP-45"
-      }
-    }
-  },
-  "connectwallet": {
-    "sections": {
-      "header": "Cardano Connect",
-      "connectbtn": "Connect with Cardano"
-    },
-    "connectwalletmodal": {
-      "header": "New connection",
-      "cancel": "Cancel",
-      "scanqr": "Scan QR code",
-      "pastePID": "Paste Peer ID"
-    },
-    "connectionhistory": {
-      "title": "Connection History",
-      "action": {
-        "delete": "Delete"
+    "header": "Menu",
```

Comment:
Just looking at the other tabs like Identifiers and Credentials, this is what they have in place:

```
"credentials": {
    "tab": {
      "title": "Credentials",
      "create": "Add a credential",
      "viewarchived": "View archived credentials",
      "favourites": "Favourites",
      "allcreds": "All credentials"
    },
    "layout": {
      "issued": "Issued"
    },
    "details": {
...
```

So for the Menu the correct nesting should actually be something like this:

```
  "menu": {
    "tab": {
      "title": "Menu",
      "items": {
        "profile": {
          "title": "Profile"
        },
        "crypto": {
          "title": "Crypto"
        },
        "connections": {
          "title": "Connections"
        },
        "connectwallet": {
          "title": "Connect Wallet",
          "cip": "CIP-45",
          "content": {
            "title": "Cardano Connect",
            "connectbtn": "Connect with Cardano",
            ...
          }
          ...
```